code/go/0chain.net/smartcontract/dbs/event/eventdb.go

```diff
@@ -0,0 +1,36 @@
+package event
+
+import (
+	"0chain.net/smartcontract/dbs"
+	"0chain.net/smartcontract/dbs/postgresql"
+)
+
+func NewEventDb(config dbs.DbAccess) (*EventDb, error) {
+	db, err := postgresql.GetPostgresSqlDb(config)
+	if err != nil {
+		return nil, err
+	}
+	eventDb := &EventDb{
+		Store: db,
+	}
+
+	if err := eventDb.AutoMigrate(); err != nil {
+		return nil, err
+	}
+	return eventDb, nil
+}
+
+type EventDb struct {
+	dbs.Store
+}
+
+func (edb *EventDb) AutoMigrate() error {
+	if err := edb.Store.Get().AutoMigrate(&Event{}); err != nil {
```

Comment:
AutoMigrate accept few income params, it can be AutoMigrate(&Event{}, &Blobber{})